base-lite/Dockerfile.ubi8

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright 2017 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ARG UBI_MINIMAL_VERSION="latest"
+FROM registry.access.redhat.com/ubi8/ubi-minimal:${UBI_MINIMAL_VERSION}
```

Comment:
Multiple differences:
This dockerfile doesn't have some yum packages, jdk and, python3 which are all present in the existing base image. Hence, we couldn't re-use.

Reply:
@elismaga with these changes, we got reduced image size without impacting functionality. We have plans to stabilise this image and perform thorough testing. Post that, we may replace the docker file of base image with this one. We don't want to keep duplicated code at multiple places.

Reply:
> Why not just use this lite image to replace the existing base image?

This lite image cannot cater the current docker images because current docker images require python for running some checks. We lite image does not have python.
Hence the current docker images are not compatible with the base-lite.

Reply:
1. Maybe you should be using layers then since this dockerfile and the existing base image dockerfile are so similar. This isn't a requirement, but a suggestion that may help out. This relates to other comments I made in this PR about it looking like files were copied around. Using layers may help avoid that issue.
2. If you don't have the JDK installed how are you going to run any of the CP services in these images since they are all Java?
3. The reason you have PR where you re-wrote cub/dub in bash just so you don't have to install Python here?

Reply:
>If you don't have the JDK installed how are you going to run any of the CP services in these images since they are all Java?
We have removed the JDK but kept the JRE

>The reason you have PR where you re-wrote cub/dub in bash just so you don't have to install Python here?
We have removed Python, JDK(kept only JRE) and some other packages which are not needed for this lite image